runtime/compiler/x/runtime/X86Codert.nasm

```diff
@@ -487,8 +487,9 @@ _dblRemain:
         push _rax
         push _rbx
         push _rcx
-        sub _rsp, 48                              ; _rsp = _rsp - 48(bytes)
+        sub _rsp, 56                              ; _rsp = _rsp - 56(bytes)
```

Comment:
Sorry for the basic question, but is it necessary to change the size here, and in other places in this function, from 48 to 56 bytes?  It looks like nothing is ever stored from offset 48 upwards, and xmm4 itself is stored at offset 24.

If this really is necessary, then I believe the [code under the `LARGE_NUMS:` case](https://github.com/eclipse-openj9/openj9/blob/aaa589e463a92685205b4cd330e59d4cdef22032/runtime/compiler/x/runtime/X86Codert.nasm#L626-L639) below would also need to add 56 to rsp on return.

Reply:
Yes, you are right